wof/wofpy_config.py

```diff
@@ -14,22 +16,25 @@
 
 Usage:
     wofpy_config INDIR
+    wofpy_config INDIR [--mode=<test,production>]
 
     wofpy_config (-h | --help | -v | --version)
 
 Examples:
-    wofpy_config my_wofpy_config
+    wofpy_config wofpyserver
```

Comment:
> Actually, we should accommodate the case of a user running wofpy_config on a **base** directory that already exists, to add another DAO. It should not fail or overwrite the base directory; failure or overwriting could happen for the DAO directory if it exists, though (eg odm2timeseries). See [my earlier comments](https://github.com/ODM2/WOFpy/issues/132#issuecomment-304333740)

Right now we are not officially shipping other DAOs, right? If so I'd rather implement this once the needs comes b/c that will entail a more complex set of options and so we don't overwrite user data.